src/pages/index.tsx

```diff
@@ -7,41 +7,36 @@ import { ArrowIcon } from "../../public/assets/icons";
 import {
   getActiveApplications,
   getActiveApplicationsPagination,
-  getGlobalContent,
 } from "../../util/client";
 import { PaginationType, Data } from "../../util/type";
 import { getLocationFromPostcode } from "../../util/geolocation";
-import { ContextApplication } from "@/context";
 import Link from "next/link";
+import { ContextApplication } from "@/context";
 
 export const itemsPerPage = 6;
 
 export async function getStaticProps() {
   const dataId = await getActiveApplications();
   const data = await getActiveApplicationsPagination({ itemsPerPage });
-  const globalContent = await getGlobalContent();
 
   return {
     props: {
       dataId,
       data,
-      globalContent,
     },
   };
 }
 
-const Home = ({ dataId, data, globalContent }: PaginationType) => {
-  const { setGlobalInfo } = useContext(ContextApplication);
+const Home = ({ dataId, data }: PaginationType) => {
+  const { globalInfo } = useContext(ContextApplication);
```

Comment:
I wonder if globalConfig makes more sense as var name. Its not just information, it configures how the site looks.
What you think?